keyboards/sofle_pico/keyboard.json

```
@@ -0,0 +1,251 @@
+{
+    "manufacturer": "Ryan Neff",
+    "keyboard_name": "Sofle Pico",
+    "maintainer": "Ryan Neff <github@JellyTitan.com>",
+    "bootloader": "rp2040",
+    "bootloader_instructions": "Hold the BOOT button while plugging in the usb cable",
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP14", "pin_b": "GP15"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "dynamic_tapping_term": true,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "oled": true,
+        "rgb_matrix": true,
+        "wpm": true
+    },
+    "matrix_pins": {
+        "cols": ["GP1", "GP2", "GP3", "GP4", "GP5", "GP8"],
+        "rows": ["GP9", "GP10", "GP11", "GP12", "GP13"]
+    },
+    "mouse_key": {
+        "delay": 10,
+        "interval": 16,
+        "max_speed": 2,
+        "time_to_max": 40
+    },
+    "processor": "RP2040",
+    "rgb_matrix": {
+        "sleep": true,
+        "animations": {
+            "band_pinwheel_sat": true,
+            "band_pinwheel_val": true,
+            "band_sat": true,
+            "band_spiral_sat": true,
+            "band_spiral_val": true,
+            "band_val": true,
+            "breathing": true,
+            "cycle_all": true,
+            "cycle_left_right": true,
+            "cycle_out_in": true,
+            "cycle_out_in_dual": true,
+            "cycle_pinwheel": true,
+            "cycle_spiral": true,
+            "cycle_up_down": true,
+            "digital_rain": true,
+            "dual_beacon": true,
+            "gradient_left_right": true,
+            "gradient_up_down": true,
+            "hue_breathing": true,
+            "hue_pendulum": true,
+            "hue_wave": true,
+            "jellybean_raindrops": true,
+            "multisplash": true,
+            "pixel_flow": true,
+            "pixel_rain": true,
+            "rainbow_beacon": true,
+            "rainbow_moving_chevron": true,
+            "rainbow_pinwheels": true,
+            "raindrops": true,
+            "solid_multisplash": true,
+            "solid_reactive": true,
+            "solid_reactive_cross": true,
+            "solid_reactive_multicross": true,
+            "solid_reactive_multinexus": true,
+            "solid_reactive_multiwide": true,
+            "solid_reactive_nexus": true,
+            "solid_reactive_simple": true,
+            "solid_reactive_wide": true,
+            "solid_splash": true,
+            "splash": true,
+            "typing_heatmap": true
+        },
+        "default": {
+            "animation": "cycle_out_in",
```

Not required, but given that this is a split, the dual animation looks better on splits...
```suggestion
            "animation": "cycle_out_in_dual",
```